public/docs/ts/latest/guide/structural-directives.jade

```diff
@@ -40,13 +39,16 @@ a#definition
   The directive then does whatever it's supposed to do with that host element and its descendents.
 
   Structural directives are easy to recognize. 
-  An asterisk (\*) precedes the directive attribute name as in this example.
-+makeExample('structural-directives/ts/src/app/app.component.html', 'ngif')(format=".")
+  An asterisk (*) precedes the directive attribute name as in this example.
+
++makeExcerpt('src/app/app.component.html', 'ngif', '')
```

Comment:
Yes, this is shorter and might be worth while if we were staying with Jade.. But we're converting all of this anyway and this kind of move is more of a distraction than a help right now.

I'll accept in THIS PR but please don't do this on future PRs.